src/components/AppShell.tsx

```diff
@@ -251,19 +251,7 @@ function AppShell({ children }: AppShellParams) {
                       caption={'Xenium'}
                       path={'/lab/probe_hybridisation_xenium'}
                       icon={<LabwareIcon className="flex-shrink-0 h-6 w-6 text-sdb-400" />}
-                      description={'Recording Xenium process'}
-                    />
-                  </Menu>
-                  <Menu
-                    caption={'Xenium'}
-                    icon={<LabwareIcon className="flex-shrink-0 h-6 w-6 text-sdb-400" />}
-                    description={'Recording Probe hybridisation'}
-                  >
-                    <NavLinkMenuItem
-                      caption={'Xenium Analyser'}
-                      path={'/lab/xenium_analyser'}
-                      icon={<LabwareIcon className="flex-shrink-0 h-6 w-6 text-sdb-400" />}
-                      description={'Recording Xenium analyser information'}
+                      description={'Recording Probe hybridisation for Xenium slides.'}
```

Comment:
Again merge - The previous description was correct 'Recording Xenium analyser information'